README.md

```diff
@@ -32,7 +32,8 @@ composer run-script install
 
 ## Running
 ```
-composer run-script start
+cd docroot
+php core/scripts/drupal server
```

Comment:
Is it possible to just do `php docroot/core/scripts/drupal server` ?

Reply:
nope, this doesn't work :(

Reply:
```$ php docroot/core/scripts/drupal server

In Connection.php line 120:

  SQLSTATE[HY000] [14] unable to open database file


In Connection.php line 116:

  SQLSTATE[HY000] [14] unable to open database file


server [--host [HOST]] [--port [PORT]] [-s|--suppress-login]
```